src/screens/SurveyDetails/index.jsx

```diff
@@ -0,0 +1,36 @@
+// This class is for projection of the component
+import React from 'react';
+
+import RatingQuestion from '../../components/RatingQuestion';
+
+export class Question extends React.Component {
+  render() {
+    const question = 'How fulfilled did you feel during this WFH period?';
+
+    return (
+      <div className="questions">
+        <div className="questions__header">
+          <div className="questions__header-right">
+            <button className="button button--close">x</button>
```

Comment:
How about using the SVG icon from Figma instead of `x` 🙌 
![Screen Shot 2563-11-11 at 15 39 15](https://user-images.githubusercontent.com/1772999/98788977-2e61e900-2434-11eb-9e29-373584166b86.png)